client/dashboard/me/billing-purchases/cancel-purchase/cancel-purchase-form/step-components/upsell-step.tsx

```diff
@@ -292,23 +292,13 @@ export default function UpsellStep( {
 					image={ imgSwitchPlan }
 				>
 					<>
-						{ hasEnTranslation(
-							'%(plan)s still gives you access to fast support, removal of ads, and more — and for 50% of the cost of your current plan.'
-						)
-							? sprintf(
-									/* Translators: %(plan)s is WordPress.com Personal or another plan */
-									__(
-										'%(plan)s still gives you access to fast support, removal of ads, and more — and for 50% of the cost of your current plan.'
-									),
-									{ plan: personalPlanName }
-							  )
-							: sprintf(
-									/* Translators: %(plan)s is WordPress.com Personal or another plan */
-									__(
-										'%(plan)s still gives you access to customer support via email, removal of ads, and more — and for 50% of the cost of your current plan.'
-									),
-									{ plan: personalPlanName }
-							  ) }{ ' ' }
+						{ sprintf(
+							/* Translators: %(plan)s is WordPress.com Personal or another plan */
```

Comment:
nit: I think we always use "t"ranslators

Reply:
Thanks! I'll make this change.